pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -30,7 +30,7 @@
     <PrereleaseLibraryPackage Include="System.Data.OleDb" />
     <PrereleaseLibraryPackage Include="System.Data.SqlClient"> 
       <!-- SQL client defines its own version, higher than rest of master --> 
-      <Version>4.8.3</Version>
+      <Version>4.8.4</Version>
```

Comment:
Ok, according to @ericstj [in the old PR](https://github.com/dotnet/corefx/pull/42952#issuecomment-658533322), it is expected to bump the version of Microsoft.Windows.Compatibility too. I assume it's also expected to do it here as well. But you're missing the changes in lines 6 and 7 of this file.